.ruff.toml

```diff
@@ -0,0 +1,7 @@
+line-length = 100
+
+[lint]
+select = ["F", "I", "UP"]
```

Comment:
@jjaquish @LightOfHeaven1994 @RonnyPfannschmidt any other ruff linters that you'd like to include at this time?

Reply:
@mshriver these look good to me :+1: